tests/Domain/Purchase/Entity/SubscriptionPurchaseTest.php

```diff
@@ -17,7 +17,7 @@ final class SubscriptionPurchaseTest extends TestCase
     /** @test */
     public function instantiation(): void
     {
-        $data = [
+        $partialData = [
```

Comment:
I want to maintain the AAA pattern in each test case. Each test case should have a single Arrange section, a single Action entry point, and a single Assert section. I’ll split this test case myself since I’m already late reviewing this PR.